src/Player.ts

```diff
@@ -1996,7 +1994,7 @@ export class Player implements ISerializable<SerializedPlayer, Player> {
       if (corporationCard !== undefined &&
             corporationCard.initialAction !== undefined &&
             corporationCard.initialActionText !== undefined &&
-            this.corporationInitialActionDone === false
+            !this.corporationInitialActionDone
```

Comment:
I won't ask to change when the code is already using an exclamation mark but in cases it isn't, I'd prefer if you left it as `=== false` as it is very easy to miss the exclamation mark when reading code.

Reply:
I'll go one step further -- please restore this. I'm fairly certain I've been asked not to use that mechanism.

Reply:
Ok, that's a good point.
```suggestion
            this.corporationInitialActionDone === false
```